spec/spec.go

```diff
@@ -567,6 +577,11 @@ func (c *APISpecification) processMethod(api *APIGroup, pathItem *spec.PathItem,
 		navigationName = o.Summary
 	}
 
+	sortOrder := o.Summary
```

Comment:
This doesn't work as expected if you don't define  `x-navigateMethodsByName: true` (or set it to `false`), as the entries are still ordered by Summary, it it probably should be ordered by operationName (I tried this out on the standard petstore example which isn't really REST, but was good enough to see the issue).